requirements.txt

```diff
@@ -186,3 +187,5 @@ datadog==0.5.0
 freezegun==0.3.5
 # sha256: tYAUqlZA40Vup9vzPxUaHEud9HKJdocwwKAwYaRxOGU
 newrelic==2.56.0.42
+# sha256: VaXcePenQqDnVmRRNP-zm74R2g_qK8D3Bw1A2sIItzI
+contextlib2==0.4.0
```

Comment:
Because of https://github.com/getsentry/raven-python/blob/f9060ccf67d8b19a7d06946895b1039f65e67b53/setup.py#L28